assets/data/effects/drauven/drauven_pc_fissureBell.json

```diff
@@ -25,7 +26,7 @@
 		"template": "ANY",
 		"type": "SCENERY",
 		"choose": "BY_SCORE",
-		"scoreFunction": "((metal+rock)+furniture)+statue",
```

Comment:
Will our Deeven characters still be able to cast spells from all types of terrain with what we've taken out? (ie, can they still cast from tree, tools, furniture, book & wood if the target isn't also rock, statue, plant or textile?)

Reply:
Yes, assuming Drauven are still setup with the same Mystic class aspect to give them Interfusion, there just needs to be a way to attach drauvenInterfuse to them (or all of the effect, at the very least), and then they'll overwrite the appropriate abilities while leaving the other ones still usable.